rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -127,7 +131,15 @@ public abstract class SystemProperties {
     protected SystemProperties(ConfigLoader loader) {
         try {
             this.configFromFiles = loader.getConfig();
-            if(logger.isTraceEnabled()){
+            if (isSystemProfilingEnabled(this.configFromFiles)) {
```

Comment:
minor thing but I wold extract this profiling specific logic to a initProfiling (kind of) method. Just to avoid this constructor to grow in the future with similar new features.